src/test/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDepositSpec.scala

```diff
@@ -1139,17 +1139,74 @@ class AddDatasetMetadataToDepositSpec extends UnitSpec with BeforeAndAfterAll {
     verify(<ddm>{AddDatasetMetadataToDeposit.createMetadata(dataset)}</ddm>, expectedXml)
   }
 
-  it should "return the expected streaming surrogate relation" in {
+  "createSurrogateRelation" should "return the expected streaming surrogate relation" in {
     val dataset = new Dataset() +=
       "SF_DOMAIN" -> List("randomdomainname") +=
       "SF_USER" -> List("randomusername") +=
       "SF_COLLECTION" -> List("randomcollectionname")
-    val expectedXml = <ddm>
-      <ddm:dcmiMetadata>
-        <ddm:relation scheme="STREAMING_SURROGATE_RELATION">/domain/randomdomainname/user/randomusername/collection/randomcollectionname/presentation/$sdo-id</ddm:relation>
-      </ddm:dcmiMetadata>
-    </ddm>
-    verify(<ddm>{AddDatasetMetadataToDeposit.createMetadata(dataset)}</ddm>, expectedXml)
+    val expectedXml = <ddm:relation scheme="STREAMING_SURROGATE_RELATION">/domain/randomdomainname/user/randomusername/collection/randomcollectionname/presentation/$sdo-id</ddm:relation>
+
+    inside(AddDatasetMetadataToDeposit.createSurrogateRelation(dataset)) {
+      case Some(xml) => verify(xml, expectedXml)
+    }
+  }
+
+  it should "not return a path when the given an empty dataset" in {
+    val dataset = new Dataset()
+    AddDatasetMetadataToDeposit.createSurrogateRelation(dataset) shouldBe empty
+  }
+
+  it should "return a path with the default domain when no domain is specified" in {
+    val dataset = new Dataset() +=
+      "SF_USER" -> List("randomusername") +=
+      "SF_COLLECTION" -> List("randomcollectionname")
+    val expectedXml = <ddm:relation scheme="STREAMING_SURROGATE_RELATION">/domain/dans/user/randomusername/collection/randomcollectionname/presentation/$sdo-id</ddm:relation>
```

Comment:
it should indeed. but since I copied the place where I have the default encoded, I put a TODO for later use, since the desired refactoring is far outside the scope of this issue.